tests/unit/test_response/test_sse.py

```diff
@@ -99,3 +99,29 @@ async def numbers() -> AsyncIterator[SSEData]:
 def test_invalid_content_type_raises() -> None:
     with pytest.raises(ImproperlyConfiguredException):
         ServerSentEvent(content=object())  # type: ignore[arg-type]
+
+
+async def test_sse_ping_events() -> None:
+    @get("/test_ping")
+    async def handler() -> ServerSentEvent:
+        async def slow_generator() -> AsyncIterator[SSEData]:
+            for i in range(1):
+                await anyio.sleep(1)
```

Comment:
I would propose using smaller sleep intervals for tests. We can just decrease `ping_interval`, but tests would become slower because of this.